_posts/2019-03-26-read-models-a-knot-to-untie.md

```diff
@@ -13,7 +13,7 @@ category: view-model-composition
 redirect_from: "/2019/03/26/read-models-a-knot-to-untie.html"
 ---
 
-When dealing with distributed systems it's very tempting, and apparently easy and simple, to solve the "we need to present data to users" problem building read models. Read models designed to satisfy presentation requirements, containing data coming from different services.
+When dealing with distributed systems it's very tempting, and apparently easy and simple, to solve the "we need to present data to users" problem by building read models. Read models are designed to satisfy presentation requirements, containing data coming from different services.
```

Comment:
@kbaley As I read it, it sounds like it has a different meaning compared to my original intentions.
The second sentence is referred to the previous one, it specifies that the mentioned read models are designed to compose data coming from different services. The “are” turns it into a general statement, and it should not be.